esmvaltool/diag_scripts/climate_patterns/climate_patterns.py

```diff
@@ -0,0 +1,658 @@
+# (C) Crown Copyright 2022-2024, Met Office.
+"""Diagnostic script to build climate patterns from CMIP6 models.
+
+Description
+-----------
+Builds patterns, anomaly and climatology cubes from CMIP6 models.
+This diagnostic needs preprocessed mean monthly cubes, with no
+gridding requirements. Default re-grid specification exists to
+decrease CPU-load and run-time.
+
+Author
+------
+Gregory Munday (Met Office, UK)
+
+
+Configuration options in recipe
+-------------------------------
+jules_mode: bool, optional (default: false)
+    options: true, false
+    def: outputs extra data (anomaly, climatology) per variable
+         to drive JULES-IMOGEN configuration
+parallelise: bool, optional (default: false)
+    options: true, false
+    def: parallelises code to run N models at once
```

Comment:
In the recipe (if I'm understanding this correctly) the default values are `true` rather than `false`, correct?

Reply:
Ah yes, this is correct. I'll push the recipe file up it set as `false`

Reply:
Resolved in latest commit

Reply:
Should `parallelise` be `false` in the recipe too? 😊